pkg/plugins/policies/meshtls/plugin/v1alpha1/testdata/permissive-no-mtls.policy.yaml

```diff
@@ -0,0 +1,12 @@
+targetRef:
+  kind: Mesh
+from:
+  - targetRef:
+      kind: Mesh
+    default:
+      tlsVersion:
+        min: TLS12
+        max: TLS13
+      tlsCiphers:
+        - "ECDHE-ECDSA-AES128-GCM-SHA256"
```

Comment:
This doesn't show up in the golden file I dont think

Reply:
Because this test has mTLS disabled

Reply:
ah ok, disabled on the Mesh level